tla-types/src/main/scala/at/forsyte/apalache/tla/types/TlaType1TagPrinter.scala

```diff
@@ -0,0 +1,20 @@
+package at.forsyte.apalache.tla.types
+
+import at.forsyte.apalache.tla.lir.{TypeTag, Typed, Untyped}
+import at.forsyte.apalache.tla.lir.io.TypeTagPrinter
+import at.forsyte.apalache.tla.typecheck.TlaType1
+
+class TlaType1TagPrinter extends TypeTagPrinter {
+  def apply(tag: TypeTag): String = {
+    tag match {
+      case Untyped()           => "Untyped"
+      case Typed(tt: TlaType1) => tt.toString
+      // unexpected type, output as much as we can
+      case Typed(_) => "Typed[%s](%s)".format(tag.getClass.getSimpleName, tag)
+    }
+  }
+}
+
+object TlaType1PrinterPredefs {
+  implicit val printer: TypeTagPrinter = new TlaType1TagPrinter
+}
```

Comment:
Did you not see ` tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/TlaType1Printer.scala` ? It was written as a placeholder, so please either delete it or change it to look like this new file, so we don't have duplicate printers in the codebase.

Reply:
I would prefer to remove `TlaType1Printer`, as it is more ad-hoc. We should in general avoid objects like this one.

Reply:
Feel free to delete it then.